packages/@aws-cdk/aws-stepfunctions-tasks/package.json

```diff
@@ -129,6 +129,51 @@
   "stability": "experimental",
   "awslint": {
     "exclude": [
+      "props-default-doc:@aws-cdk/aws-stepfunctions-tasks.SagemakerUpdateEndpointTaskProps.retainAllVariantProperties",
```

Comment:
Please address all of these exclusions. The general guidance is no new exclusions should be added. It should be rare that we add one here.